invesalius/gui/task_navigator.py

```diff
@@ -1442,7 +1442,7 @@ def UpdateCurrentCoord(self, position):
 
     def UpdateNavigationStatus(self, nav_status, vis_status):
         if not nav_status:
-            self.current_coord[3:] = None, None, None
+            self.current_coord = list([None, None, None])
```

Comment:
list is not needed. Just [None, None, None] is enough.

Reply:
I did the change, thanks!